week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,16 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var precisePercentageStudents =
+  (numberOfStudents / (numberOfStudents + numberOfMentors)) * 100;
+var percentageStudents = Math.round(precisePercentageStudents);
+
+var precisePercentageMentors =
+  (numberOfMentors / (numberOfMentors + numberOfStudents)) * 100;
+var precisePercentageMentors = Math.round(precisePercentageMentors);
+
+console.log(
+  "percentage students: " + Math.round(precisePercentageStudents) + "%"
+);
+console.log(
+  "percentage mentors: " + Math.round(precisePercentageMentors) + "%"
+);
```

Comment:
in the previous exercise you created a variable total to get the sum of students and mentors, can you use this variable  in precisePercentageStudents and precisePercentageMentors, so you need to remove (numberOfMentors + numberOfStudents) and use total variable, it will reduce your code and make it cleaner